test/crud_tests.py

```diff
@@ -3,25 +3,34 @@
 """End to end test.
 """
 
-import logging
-import unittest
 import json
 import os.path
+import unittest
+import sys
 
-import pydocumentdb.documents as documents
+try:
+    # Python2.7
+    from __builtin__ import *
+except ImportError:
+    # Python3
+    import builtins
```

Comment:
What's the need for these imports?

If they are needed, should Python 3 be `from builtins import *` to match what's done for Python 2?